lib/samples/display_ogc_api_collection/README.md

```diff
@@ -0,0 +1,43 @@
+# Display OGC API collection
+
+Display an OGC API feature collection and query features while navigating the map view.
+
+![Image of display OGC API collection](display_ogc_api_collection.png)
+
+## Use case
+
+When panning the map view, it may be necessary to query the OGC API feature table for additional features within the new visible extent.
+
+## How to use the sample
+
+Pan the map and observe how new features are loaded from the OGC API feature service.
+
+## How it works
+
+1. Create an `OgcFeatureCollectionTable` object using a URL to an OGC API feature service and a collection ID.
+2. Set the feature table's `FeatureRequestMode` property to `FeatureRequestMode.manualCache`.
+3. Call `OgcFeatureCollectionTable.load()`.
+4. Create a `FeatureLayer` using the feature table and add it to the map view.
+5. Every time the map view navigation completes:
+   1. Create `QueryParameters`.
+   2. Set the parameter's `Geometry` to the current extent of the map view.
+   3. Set the parameter's `SpatialRelationship` property to `SpatialRelationship.intersects`.
+   4. Set the `maxFeatures` property to 5000 (some services have a low default value for maximum features).
+   5. Call `OgcFeatureCollectionTable.PopulateFromService()` using the query parameters from the previous steps.
```

Comment:
"PopulateFromService()" should be "populateFromService()" -- initial lowercase letter.

I think the various references to properties should be initial-lowercase as well -- featureRequestMode, geometry, spatialRelationship. It's a bit picky, but when referring to the property it should be lowercase, and when referring to the type it should be uppercase.